.github/workflows/e2e-tests.yml

```diff
@@ -48,8 +48,8 @@ jobs:
         with:
           m2-cache-key: e2e-tests
 
-      - name: Build uberjar with ./bin/build
-        run: ./bin/build version translations frontend licenses drivers uberjar
```

Comment:
There are only six steps you can run, and this lists all six. It's exactly the same as running without specifying any steps

On that note I'm not convinced we need to be doing all these steps and building the entire uberjar and all drivers and **checking licenses** in the 4 or 5 different e2e tests we do it in, but auditing that stuff is outside the scope of this PR